src/main/java/me/greenleaf/Legolas.java

```diff
@@ -1,24 +1,45 @@
 package me.greenleaf;
 
+import java.util.ArrayList;
+import java.util.Random;
+
 public class Legolas {
-    private SASS_LEVELS sass;
+    private SASS_LEVEL sass;
+    private static Legolas legolasInstance;
 
-    public static void main(String[] args) {
-        Legolas.getInstance().setSass(SASS_LEVELS.LOW);
+    private Legolas() {
+        sass = SASS_LEVEL.HIGH;
     }
 
     public static Legolas getInstance() {
-        return new Legolas();
+        if (legolasInstance == null) {
+            legolasInstance = new Legolas();
+        }
+        return legolasInstance;
+    }
+
+    public String queryMind(String question) {
+        Random rand = new Random();
```

Comment:
This should be a `SecureRandom` instance, and it should be a `ThreadLocal` instance to prevent thread contention for the resource.